shared/teams/add-members-wizard/add-email.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Container from '../../util/container'
+import * as Constants from '../../constants/teams'
+import * as Styles from '../../styles'
+import {ModalTitle} from '../common'
+
+type Props = {
+  teamname: string
+  errorMessage: string
+}
+
+const AddEmail = (props: Props) => {
+  const [invitees, setInvitees] = React.useState('')
+
+  const dispatch = Container.useDispatch()
+  const nav = Container.useSafeNavigation()
+  const onBack = () => dispatch(nav.safeNavigateUpPayload())
+
+  const waiting = Container.useAnyWaiting(Constants.teamCreationWaitingKey)
```

Comment:
I don't think this will use a waiting key since hitting continue doesn't make any async call